lib/spack/spack/test/cmd/common/arguments.py

```diff
@@ -4,6 +4,7 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 import argparse
+import sys
```

Comment:
Is this required? It's the only added line so I'm surprised if it's necessary.

Reply:
It is not, it's a vestige of the skip tests on Windows. I'll remove it.